test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 3000
+
+bench_run: true
+
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=8 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=16 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=24 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=32 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=39 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=46 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=53 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
+]
+stress_cmd: ["scylla-bench -workload=uniform -mode=read -select-order-by=asc -replication-factor=3 -partition-count=60 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -rows-per-request=2000 -timeout=240s -retry-number=10 -concurrency=400 -duration=44h -connection-count 400"
+]
+
+n_db_nodes: 4
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i4i.4xlarge'
```

Comment:
When the stress finish the 2 days, check what is the disk usage (in monitor) for each node.
We may be able to make it smaller.

Reply:
will be retested with latest s-b version (not private build).

Reply:
> When the stress finish the 2 days, check what is the disk usage (in monitor) for each node. We may be able to make it smaller.
@roydahan ,
The disk usage is 5% after one day:
![image](https://user-images.githubusercontent.com/9427929/207285285-45356d20-26e6-48ed-a621-5bfd854dcb0b.png)

The CPU load is around 50 - 70%:

![image](https://user-images.githubusercontent.com/9427929/207284830-65a22b5c-cb89-4d82-aab0-d42aaf207e08.png)

Reply:
disk utilization is still around 6%

Reply:
Now the question is if we need to retest with 2xlarge or leave it as is.